api/envoy/api/v2/rds.proto

```diff
@@ -63,19 +64,24 @@ message RouteConfiguration {
   // :ref:`envoy_api_msg_route.RouteAction`. For more information, including details on
   // header value syntax, see the documentation on :ref:`custom request headers
   // <config_http_conn_man_headers_custom_request_headers>`.
-  repeated core.HeaderValueOption response_headers_to_add = 4;
+  repeated core.HeaderValueOption response_headers_to_add = 4
+      [(validate.rules).repeated .max_items = 1000];
 
   // Specifies a list of HTTP headers that should be removed from each response
   // that the connection manager encodes.
-  repeated string response_headers_to_remove = 5;
+  repeated string response_headers_to_remove = 5 [
```

Comment:
This is headers to remove. I thought we could only trigger the assert by making the headers too long?